addons/nametags/functions/fnc_onDraw3d.sqf

```diff
@@ -104,10 +69,17 @@ if (_enabledTagsNearby) then {
 
         if !(isNull _target) then {
             private _drawSoundwave = (GVAR(showSoundWaves) > 0) && {[_target] call FUNC(isSpeaking)};
-            if (_enabledTagsCursor && {!_drawSoundwave}) exitWith {}; // (Cursor Only && showSoundWaves==2) - quick exit
-
-            private _relPos = (visiblePositionASL _target) vectorDiff _camPosASL;
-            private _distance = vectorMagnitude _relPos;
+            
+            // W- disabled below
+            //if (_enabledTagsCursor && {!_drawSoundwave}) exitWith {}; 
+            // (Cursor Only && showSoundWaves==2) - quick exit
+
+            private _distance = ACE_Player distance _target;
+            
+            // W- Changed distance to be absolute distance from player to target.
+            // This prevents players seeing less tags (or fainter ones) in third person.
```

Comment:
Correct. Remote controlled units will still be able to see tags, but the Zeus camera will not. That said, tag distance by default is 5m and it's very, very infrequent for the Zeus camera to ever see tags anyways.